### ISSUE TITLE
Where-Object is not highlighted correctly

### ISSUE THREAD

**Issue by pcgeek86** [PowerShell/EditorSyntax#22]
Check out `Where-Object` in this screenshot / example.

![image](https://cloud.githubusercontent.com/assets/466713/18399405/05603ac8-768e-11e6-8ed8-1f56de79f13a.png)

```
function Clear-JunctionLinks {
    [CmdletBinding()]
    param (
    )

    Get-ChildItem -Path $env:TEMP -Directory | Where-Object -FilterScript { }
}
```

Cheers,
**Trevor Sullivan**
Docker Captain
Microsoft MVP: Cloud & Data Center Management
https://trevorsullivan.net
https://twitter.com/pcgeek86

**Comment by pcgeek86** [comment]
@gravejester I noticed you mentioned a series of "test scripts" in #16. Can you add this scenario to those?

**Comment by gravejester** [comment]
@pcgeek86 Yes, I'll add these as well. Thank you :)

**Comment by pcgeek86** [comment]
@gravejester Thank you sir -- hoping the syntax highlighting becomes world-class in VSCode! :)

**Comment by MovGP0** [comment]
see also #39

**Comment by omniomi** [comment]
Resolved as of #94
